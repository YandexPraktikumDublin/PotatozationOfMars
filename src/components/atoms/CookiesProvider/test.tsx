import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { CookiesProvider } from '.'

describe('<CookiesProvider />', () => {
  it('should renders correct <CookiesProvider />', () => {
    const wrapper = shallow(
      <CookiesProvider>
        <></>
      </CookiesProvider>
    )

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
