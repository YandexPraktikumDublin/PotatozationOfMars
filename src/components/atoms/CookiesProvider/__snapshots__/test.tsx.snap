// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CookiesProvider /> should renders correct <CookiesProvider /> 1`] = `
<ContextProvider
  value={[Function]}
/>
`;
